vpr/src/route/build_switchblocks.cpp

```diff
@@ -422,6 +432,73 @@ static bool is_core(const DeviceGrid& grid, int x, int y) {
     return is_core;
 }
 
+static bool match_sb_xy(const DeviceGrid& grid, int x, int y, int sb_x, int sb_y, const e_sb_loc_spec& sb_reg_x, const e_sb_loc_spec& sb_reg_y){
+    
+    //if one of sb_x and sb_y is defined, we either know the exact location (x,y) or the exact x location (will apply it to all columns) 
+    //or the exact y location (will apply it to all rows)   
```

Comment:
When we know the exact x location while y is not specified, doesn't this mean that we know the column and apply it all the rows?